index.html

```diff
@@ -313,6 +313,22 @@ <h2 class="primary-text-about-me">About
     </section>
     <!--End-About-me-->
   </main>
+  <footer>
+    <div class="contact-me">
+      <form class="form-contact-me" action="#" method="post">
```

Comment:
:red_circle: You have done well implementing the form. But the action attribute is empty.  [The  project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md) state that you should use the [Formspree service (Click here for more)](https://formspree.io/html) to Collect data. 


![Screenshot from 2023-01-17 19-51-06](https://user-images.githubusercontent.com/26736582/212975092-c4f09254-1e01-40da-88d1-0c30c4b91c70.png)

The action value should be similar to `https://formspree.io/f/{form_id}` . The {form_id} is just a placeholder.  you have to get your unique id and then replace it with the placeholder This will allow you to submit the form successfully.

Reply:
> Correction

*Form successfully submitted and received*

|Form|Answers|
|:---------------:|:----------------:|
|![imagen](https://user-images.githubusercontent.com/90431831/214444499-ab2f7352-60a2-4e9e-895f-bec4958b32e5.png)|![imagen](https://user-images.githubusercontent.com/90431831/214444969-6101e7f6-c3d1-4b07-8f0a-6fb420058f8a.png)